### PR TITLE
Update Core App Available Pages + Navbars

### PR DESCRIPTION
**Pull request by controtie**
# **Pull Request Template**

###### *Select a category that relates to the content of your pull request (choose all that apply).*

- [ ] Bug Fix
- [ ] Content
- [ ] New Feature
- [ ] Documentation
- [x] Code Refactoring

&nbsp;

## **Summary of Changes**

Sets up the general navigation structure of the new core application.
Updates Navbar and removes unused pages from `App.tsx`.

The following pages/links have been removed:
Home, Vote, Docs, Institutions, Rewards, About, News, How To Buy, Careers

A follow up PR will be made removing all unused pages and providers.

## **Test Data or Screenshots**

image
image
image

###### *By submitting this pull request, you are confirming the following to be true:*

- I have reviewed the [Contribution Guidelines](https://github.com/SetProtocol/index-ui/blob/master/CONTRIBUTING.md).
- I have performed a self-review of my own code.
- I have updated my repository to match the master at `SetProtocol/index-ui`.
- I have included test data or screenshots that prove my fix is effective or that my feature works.

### CODE REVIEW

File: src/components/Logo/Logo.tsx [hunk 1]
```diff
@@ -15,16 +14,14 @@ const Logo: React.FC = () => {
     logo = darkMode ? indexLogoFullWhite : indexLogoFullBlack
   else logo = darkMode ? indexLogoWhite : indexLogoBlack
 
-  console.log(window.innerWidth)
```
Comment (rootulp) [inline src/components/Logo/Logo.tsx:18]: lol same idea: https://github.com/SetProtocol/index-ui/pull/504

^ I'll close my PR in favor of this

File: src/App.tsx [hunk 2]
```diff
@@ -34,8 +34,8 @@ import { TransactionWatcherProvider } from 'contexts/TransactionWatcher'
 import { V3FarmingProvider } from 'contexts/V3Farming'
 import { BedTokenMarketDataProvider } from 'contexts/BedTokenMarketData'
 import { BedIndexComponentsProvider } from 'contexts/BedIndexComponents'
-import { StreamingFeeProvider } from "contexts/StreamingFee"
-import { TokenSupplyProvider } from "contexts/TokenSupply"
+import { StreamingFeeProvider } from 'contexts/StreamingFee'
+import { TokenSupplyProvider } from 'contexts/TokenSupply'
```
Comment (rootulp) [inline src/App.tsx:38]: heads up small merge conflict: https://github.com/SetProtocol/index-ui/pull/502 which does this across the codebase

Comment (controtie) [reply src/App.tsx:38]: Sounds good! I'll rebase after merging #502

Comment (0xModene) [reply src/App.tsx:38]: glhf

**Review by 0xModene — APPROVED** [review]
delet

**Pull request merged**